extensions/mathsys.py

```
@@ -20,18 +20,16 @@
 class namespace:
     # CLASS -> VARIABLES
     code: str
-    validate: bool
     # CLASS -> INIT
-    async def init(self, request: Request, response: Response) -> namespace:
-        return self
+    async def init(self, socket: æ.WebSocket) -> namespace: return self
```

Ruff (0.12.2)

Unused method argument: `socket`

(ARG002)